src/ServiceContainer/WebApiExtension.php

```diff
@@ -50,7 +50,7 @@ public function configure(ArrayNodeDefinition $builder)
         $builder
             ->addDefaultsIfNotSet()
             ->children()
-                ->scalarNode('base_url')
+                ->scalarNode('base_uri')
```

Comment:
Guzzle is changed their parameter to base_uri.
I could add some compatibility code to check for either, with preference to base_uri